src/uwtools/rocoto.py

```diff
@@ -0,0 +1,44 @@
+"""
+Support for creating Rocoto XML workflow documents.
+"""
+
+from uwtools.config import YAMLConfig
+from uwtools.j2template import J2Template
+
+# Private functions
+
+
+def _add_jobname(tree: dict) -> None:
+    """
+    Add the jobname entry for all the tasks in the workflow.
+
+    :param tree: Dict of tree  in workflow.
```

Comment:
Alternative phrasing for your consideration:

1. `Add a "jobname" attribute to each "task" element in the given config tree`.

To me, "the jobname" could mean that there's a single value instead of different values for each task; and "for all the tasks" is different than "to each task", in that the latter means we're taking action on multiple "things", rather than maybe just creating a single thing that is somehow _about_ all those things. Does that make sense?

2. `:param tree: A config tree containing "task" elements.`

By looking at the function signature, we can see that `tree` is a `dict`, so there's no need to repeat that. "workflow" is not a concept otherwise present in this function, so it might not be useful to introduce it here; instead this param tag uses the terms `task` and `element`, which also appear as concepts in the code.

What do you think? Do those make sense and would they provide any more clarity to a reader not necessarily familiar with this module and function?